Runtime/CesiumCameraController.cs

```diff
@@ -7,6 +7,10 @@
 using UnityEngine.InputSystem;
 #endif
 
+#if UNITY_IOS || UNITY_ANDROID
+using Touch = UnityEngine.InputSystem.EnhancedTouch;
```

Comment:
Can this be called `EnhancedTouch` instead? Looking at something like `Touch.Touch.activeTouches` is confusing

Reply:
Yes, thanks that is better.